scripts/list.ts

```diff
@@ -4,12 +4,12 @@
  * @param {number} n   - The number of elements in each part.
  * @returns {any[][]}
  */
-export function partition(list, n) {
-    let result = [];
+export function partition(list: any, n: number): any[][] {
```

Comment:
Avoid using the `any` type whenever you can in general - the only times this might be difficult is when creating an API but, you would ideally return an `unknown` type instead. In this case, however, you can do something like

```TS
export function paritition<Type>(list: Type[], n: number): Type[][] {
/* body */
}
```
to ensure that you have type information available on both ends. The problem with using the `any` type is that it - effectively - tells the TypeScript static type checker to not enforce any static safety checks for that function which can result in errors.